### DISCUSSION TITLE
第5节课作业(15班)

### DISCUSSION THREAD

**Discussion by ironartisan** [InternLM/Tutorial#95] [作业]
提交第5节课作业

**Comment by szhowardhuang** [comment]
![image](https://github.com/InternLM/tutorial/assets/1407300/d60ac094-39ca-4e5f-8f1f-6336ffa407e4)

**Comment by woodx9** [comment]
请参考实操部分记录[飞书链接](https://gxloh9mfvxh.feishu.cn/wiki/OvLxw1ClfinJAZkuDwzcpd9ZnZb?from=from_copylink)

**Comment by Mrguanglei** [comment]
https://gzr5xfkph4.feishu.cn/docx/AbfodCP0kohmGlxmHsVcsP5rnub?from=from_copylink

**Comment by IyGnahz** [comment]
笔记+作业：https://g6xuom3zbh.feishu.cn/docx/Ym96d6IYeocm3KxDOCnc1s1Lnac

**Comment by Mrguanglei** [comment]
https://mqf0lvyk616.feishu.cn/wiki/O1C3wmNr3iWF4nkw308cP2vCngb?from=from_copylink

**Comment by scchy** [comment]
第5节课作业: https://github.com/scchy/My_Learn/blob/master/LLM_prime/5_InternLM_Imdeploy_HW.md

**Comment by dqyydq** [comment]
https://blog.csdn.net/qq_52410386/article/details/135669415?spm=1001.2014.3001.5501

**Comment by lizhunkg** [comment]
第五课作业：https://note.youdao.com/s/YrwBlNJI

**Comment by seienn** [comment]
https://gvxxiiwsufy.feishu.cn/wiki/LG7OwhT4Pi2e5PkvjM9cXDPanSb?from=from_copylink

**Comment by sokolo05** [comment]
第五节课作业：https://github.com/sokolo05/Scholar_PuYu/blob/main/01.%E8%AF%BE%E7%A8%8B%E4%BD%9C%E4%B8%9A/%E7%AC%AC5%E8%8A%82_%E4%BD%9C%E4%B8%9A.md

**Comment by ziptoam** [comment]
https://note.youdao.com/s/D5x51QmK

**Comment by lta155** [comment]
http://t.csdnimg.cn/bvP4r

**Comment by yabgdsajghsj** [comment]
https://note.youdao.com/s/8bsjSLVo

**Comment by bblgbr** [comment]
第五节课作业：https://github.com/bblgbr/play_llm/blob/main/InterLM_Class/05%20LMDeploy%E5%A4%A7%E6%A8%A1%E5%9E%8B%E9%83%A8%E7%BD%B2-%E4%BD%9C%E4%B8%9A.md

**Comment by NANDSS1** [comment]
https://www.yuque.com/nandss/gyyg9p/gkdlk4ks9hic8ous?singleDoc# 《lesson5 LMDeploy 大模型量化部署实践（作业）》

**Comment by g29times** [comment]
https://novagen.notion.site/a822cb5aebd949acb9cde809d05e1697?pvs=4

**Comment by g29times** [comment]
https://novagen.notion.site/a822cb5aebd949acb9cde809d05e1697?pvs=4https://colab.research.google.com/drive/14y5X5j7JZ_1dDNfO80zB2gHX55cbv3mM?usp=sharing

**Comment by HarveyLiii** [comment]
## 基础作业
### 命令行本地对话
<img width="1920" alt="截屏2024-02-22 18 40 18" src="https://github.com/InternLM/tutorial/assets/127242206/6279d52f-da35-491f-87a4-e35cb37d16bb">

### API服务
<img width="1920" alt="截屏2024-02-22 18 43 48" src="https://github.com/InternLM/tutorial/assets/127242206/c5768fd3-c0a9-48ad-89ea-98d70651682e">

### 网页端
<img width="1920" alt="截屏2024-02-22 18 49 01" src="https://github.com/InternLM/tutorial/assets/127242206/270bad9c-f9ea-44e5-a2bd-17b6a5e04436">
